### PR TITLE
"what's new" errors inbox post

### PR DESCRIPTION
**Pull request by rachelsiemens**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve? This updated users to new features released by the errors inbox team. Please publish ASAP.
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by paperclypse** [comment]
Hi @rachelsiemens, thanks for your PR. It looks like you still need to sign the contributor license agreement before we can more forward with this. Thanks!

File: src/content/whats-new/2022/01/whats-new-errors-inbox-browser-mobile-1-20.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+summary: 'Triage Browser and Mobile errors without setting up a workload' 
+releaseDate: '2022-01-20' 
+getStartedLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/'
+---
+
+New Relic Errors Inbox is an error tracking solution that provides a single place to view, triage, and resolve errors across the full application stack.
+Now we have added Errors Inbox to the Browser and Mobile UIs, which makes triaging Browser and Mobile errors a breeze without the need to set up a workload.
+
+Watch an extended demo in our recent Nerdlog episode below to learn more about how errors inbox can help you track your errors intelligently:
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=M52jeJsnOfU" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (paperclypse) [inline src/content/whats-new/2022/01/whats-new-errors-inbox-browser-mobile-1-20.md:12]:
```suggestion
<Video id="M52jeJsnOfU" type="youtube" />
```

This is the format for embedding YouTube videos

File: src/content/whats-new/2022/01/whats-new-errors-inbox-browser-mobile-1-20.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+summary: 'Triage Browser and Mobile errors without setting up a workload' 
+releaseDate: '2022-01-20' 
+getStartedLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/'
+---
+
+New Relic Errors Inbox is an error tracking solution that provides a single place to view, triage, and resolve errors across the full application stack.
+Now we have added Errors Inbox to the Browser and Mobile UIs, which makes triaging Browser and Mobile errors a breeze without the need to set up a workload.
+
+Watch an extended demo in our recent Nerdlog episode below to learn more about how errors inbox can help you track your errors intelligently:
+<Video id="M52jeJsnOfU" type="youtube" />
```
Comment (roadlittledawn) [inline src/content/whats-new/2022/01/whats-new-errors-inbox-browser-mobile-1-20.md:12]: whats new posts should use `<iframe>` and not Video component. see other whats new posts as example

**Review by paperclypse — APPROVED** [review]
The embedded video is working now.

**Pull request merged**